src/common/actions/register-name.js

```diff
@@ -84,6 +84,7 @@ export function internalRegisterName(root, discharge, snapName) {
     },
     body: JSON.stringify({
       snap_name: snapName,
+      repository_url: repositoryUrl,
```

Comment:
This is OK for now, but bear in mind that `/api/store/register-name` isn't intended to stick around for the long term: once SCA sends the right CORS headers, we want to switch over to having the client talk to SCA directly.  So we probably need a separate endpoint to deal with invalidation.

Please update `docs/routes.md`.